openmetadata-service/src/main/java/org/openmetadata/service/search/indexes/TableIndex.java

```diff
@@ -46,22 +49,32 @@ public Map<String, Object> buildESDoc() {
     List<SearchSuggest> schemaSuggest = new ArrayList<>();
     List<SearchSuggest> databaseSuggest = new ArrayList<>();
     List<SearchSuggest> serviceSuggest = new ArrayList<>();
+    Set<List<TagLabel>> tagsWithChildren = new HashSet<>();
```

Comment:
it should be a single set not set of Lists

Reply:
@harshach, we are getting List<TagLable> from functions so its done that way other wise will need loops to add it.. we are flattening it before adding to doc.